islandora_bagit.module

```diff
@@ -1132,7 +1135,7 @@ function islandora_bagit_get_octetstream_sum($files) {
   $filesize_sum = 0;
   foreach ($files as $file) {
     $file_counter++;
-    $filesize_sum = filesize($file['source']) + $filesize_sum;
+    $filesize_sum = $filesize_sum + filesize($file);
   }
   return $filesize_sum . '.' . $file_counter;
```

Comment:
@mjordan question. is 0.0 a valid Payload-oxum? If you have no sources (edge case?) not even sure if possible.
Also filesize($file) returns FALSE if there is an issue (disk full, can't read, who knows). You would probably want to cast it to an (int) in that case

Reply:
@DiegoPino the Bagit spec doesn't prohibit an empty `/data` directory, and defines the Payload-Oxum as "the total number of octets (8-bit bytes) across  all payload file content and [...] the total number of payload files" so I'd say it's allowed. I haven't tested this reading of the spec against any BagIt validators however.

Thanks for finding the `filesize()` omission, I'll fix that.

Reply:
Added in 6fc5f09a22b937ecf78f6120bb7ab1514fe75148.